docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5092,6 +5092,9 @@ image](#create-an-image) section for more details.
     - **Delay** – Amount of time between updates.
     - **FailureAction** - Action to take if an updated task fails to run, or stops running during the
       update. Values are `continue` and `pause`.
+    - **Monitor** - Amount of time to monitor each updated task for failures.
+    - **MaxFailureRatio** - The fraction of tasks that may fail during an update before the
```

Comment:
Same here.

Also wondering; what's the default FailureAction (I just noticed there's no default in the description)